pkg/kubeapiserver/default_storage_factory_builder.go

```diff
@@ -125,6 +125,9 @@ func (c *completedStorageFactoryConfig) New() (*serverstorage.DefaultStorageFact
 
 		servers := strings.Split(tokens[1], ";")
 		storageFactory.SetEtcdLocation(groupResource, servers)
+
+		prefix := tokens[2]
```

Comment:
this must tolerate `len(tokens) == 2` and preserve existing behavior in that case